mta/tests/migration/controls/test_stakeholder_group.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_string
+
+
+@pytest.mark.sg
```

Comment:
I think it is ok to mark it `stakeholder_groups`

Reply:
Done. Thanks!